pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -383,6 +376,19 @@ ssize_t sock_dtls_send(sock_dtls_t *sock, sock_dtls_session_t *remote,
                       len);
 }
 
+static ssize_t _get_buffer(sock_dtls_t *sock, void *data, size_t max_len)
```

Comment:
minor: I don't like the name :P `_get_buffer` does not sound like this is copying data.

Reply:
Renamed to `_copy_buffer()`.